test/test_util.dart

```diff
@@ -7,10 +7,10 @@ import 'dart:async';
 import 'package:shelf/shelf.dart';
 
 // "hello,"
-const HELLO_BYTES = const [104, 101, 108, 108, 111, 44];
+const helloBytes = const [104, 101, 108, 108, 111, 44];
```

Comment:
https://www.dartlang.org/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names 😄